x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/overview/overview/monitor_detail_flyout.tsx

```diff
@@ -0,0 +1,460 @@
+                  name: 'Previous period',
+                },
+              ]}
+            />
+            <EuiSpacer />
+            <EuiTitle size="s">
+              <h3>{MONITOR_DETAILS_HEADER_TEXT}</h3>
+            </EuiTitle>
+            <EuiSpacer />
+            <BodyInfo
+              header={LAST_RUN_HEADER_TEXT}
+              content={
+                monitorDetail.data?.timestamp ? (
+                  <Time timestamp={monitorDetail.data?.timestamp} />
+                ) : (
+                  ''
+                )
+              }
+            />
+            <BodyInfo
+              header={LAST_MODIFIED_HEADER_TEXT}
+              content={
+                monitorSavedObject.updated_at ? (
+                  <Time timestamp={monitorSavedObject.updated_at} />
+                ) : (
+                  ''
+                )
+              }
+            />
+            <BodyInfo header={MONITOR_ID_ITEM_TEXT} content={props.id} />
+            <BodyInfo
+              header={MONITOR_TYPE_HEADER_TEXT}
+              content={capitalize(monitorSavedObject.type)}
+            />
+            <BodyInfo
+              header={FREQUENCY_HEADER_TEXT}
+              content={freqeuncyStr(monitorSavedObject?.attributes.schedule)}
+            />
+            <BodyInfo
+              header={TAGS_HEADER_TEXT}
+              content={
+                <>
+                  {monitorSavedObject?.attributes.tags?.map((tag) => (
+                    <EuiFlexItem key={`${tag}-tag`} grow={false}>
+                      <EuiBadge color="hollow">{tag}</EuiBadge>
+                    </EuiFlexItem>
+                  ))}
+                </>
+              }
+            />
+            <BodyInfo
+              header={URL_HEADER_TEXT}
+              content={
+                monitorDetail.data?.url?.full ? (
+                  <EuiLink href={monitorDetail.data.url.full}>
+                    {monitorDetail.data.url.full}
+                  </EuiLink>
+                ) : (
+                  ''
+                )
+              }
+            />
+          </EuiFlyoutBody>
+          <EuiFlyoutFooter>
+            <EuiFlexGroup justifyContent="spaceBetween">
+              <EuiFlexItem grow={false}>
+                <EuiButtonEmpty>{CLOSE_FLYOUT_TEXT}</EuiButtonEmpty>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButton
+                  // `detailLink` can be undefined
+                  isDisabled={!detailLink}
+                  fill
+                  href={detailLink}
+                  iconType="sortRight"
+                  iconSide="right"
+                >
+                  {GO_TO_MONITOR_LINK_TEXT}
+                </EuiButton>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlyoutFooter>
+        </>
+      )}
+    </EuiFlyout>
+  );
+}
+
+function BodyInfo({ header, content }: { header: string; content: JSX.Element | string }) {
```

Comment:
The markup of this component should be using `EuiDescriptionList` and `EuiDescriptionListItem`s. Use `type="column"` and `compressed={true}` and `gutterSize="s"` for description list.

Reply:
We're currently blocked for the `gutterSize` prop [until EUI 66 lands in Kibana](https://github.com/elastic/kibana/pull/141279). I'll keep a pin in this and address if we end up still not being able to set this prop.